engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglInput.java

```diff
@@ -47,36 +44,25 @@ public void postInitialise(Context rootContext) {
         this.context = rootContext;
         initControls();
         updateInputConfig();
-        Mouse.setGrabbed(false);
+        // Mouse.setGrabbed(false);
```

Comment:
Eh? Also this reminds me about the rare issues where the mouse cursor gets grabbed funny and won't let go for some devs. No idea if that's related, like if you had commented this out to avoid that while working on the PR?